### DISCUSSION TITLE
Justpy contribution policy

### DISCUSSION THREAD

**Discussion by sandeep-gh** [elimintz/justpy#623] [General]
What is policy for contribution to Justpy. I am unable to contribute directly to justpy, i.e., I have submitted patches but  I  don't show up as one of the contributors for justpy in Github. Is there guideline who can be direct contributors. Some clarification would help me plan my projects and effort accordingly and would also be beneficial for other who would like to contribute.

**Comment by WolfgangFahl** [comment]
I have no clue how the contribution display is done by github. I had expected that if you supply a patch you'd show up. I can invite you so that you can directly commit to the repository which would obviously give you greater responsibility because you'd have to try hard not to break the CI. See also https://github.com/zauberzeug/nicegui/issues/168 for an example of the downstream effects we would have to take into account. Today i talked to Eli and i assume it would be good if we have more community online meetings to have better exchange of ideas. May be we can also to an online/hybrid hackathon in the upcoming weeks. I'd love to see the collaboration to increase.

**Comment by WolfgangFahl** [comment]
_"Contributions to master, excluding merge commits and bot accounts"_ is what i see. And https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile seems to have explanations.

**Comment by sandeep-gh** [comment]
I am also not well versed with github workings, Thanks for link. I think I have an idea what could be wrong. Will try to see if it is fixable. 

+1 for  more community online meetings. But I would suggest to collect an agenda of topics that we needs to discussed.  
Also, hackathon sounds good idea, if there is an important pressing issues that need resolution.

**Comment by WolfgangFahl** [comment]
@sandeep-gh  I won't be able to organiz anything before mid december/beginning of january. If you'd like to have a meeting earlier please take the lead.
https://github.com/all-contributors/all-contributors seems to be a nice contributors visibility approach see https://github.com/bisohns/search-engine-parser for an example
